tests/sentry/monitors/test_tasks.py

```diff
@@ -106,16 +106,19 @@ def test_missing_checkin(self, mark_environment_missing_mock):
         # last_checkin was NOT updated. We only update this for real user check-ins.
         assert monitor_environment.last_checkin == ts - timedelta(minutes=2)
 
+        # next_checkin IS updated for when we're expecting the next checkin
+        assert monitor_environment.next_checkin == ts
```

Comment:
This wasn't being checked before, this is that scenario where it would be a minute later than it should have been.